### DISCUSSION TITLE
How to send an email to confirm the email address?

### DISCUSSION THREAD

**Discussion by moroshko** [supabase/supabase#11360] [Questions]
I'm using `@supabase/auth-helpers-nextjs` and signing up users using:
```
const { data, error } = await browserSupabaseClient.auth.signUp({ email, password });
```

This sends an email to the user with a link to confirm their email address.

Let's say that, the user, for whatever reason, didn't get this email.

How could I resend this email?

**Comment by logemann** [comment]
see https://github.com/supabase/supabase/discussions/3526
